master_duff.py

```diff
@@ -165,6 +174,10 @@ async def produceStats():
 
             # To see what the format of this JSON is, please check README.md
             elo_info = message.embeds[0].to_dict()
+            # PRINT MESSAGE FOR TESTING ONLY
+            print(elo_info)
+
+            # FIELDS ATTRIBUTE NO LONGER EXISTS, IGNORE FOR V2
             if "fields" not in elo_info.keys():
```

Comment:
I'm pretty sure it's this, the new format doesn't have `fields` anymore, so make it work with new or old style

Reply:
```suggestion
            if "fields" not in elo_info.keys() and "description" not in elo_info,keys():
```